provider/tests/provider_test.go

```diff
@@ -331,11 +313,12 @@ func TestRemoteCommandStdoutStderrFlag(t *testing.T) {
 	})
 
 	t.Run("update-actual-without-std", func(t *testing.T) {
+		t.Parallel()
```

Comment:
Both of these tests update the same resource, right? Can they really run in any order then?

Reply:
Huh, I apparently only read half of your [comment](https://github.com/pulumi/pulumi-command/pull/451#discussion_r1636953415), and just made them run in parallel…

I didn't have any issues with the parallelism when running the tests locally, but I suppose you're right that they might get in each other's way.

Addressing your original comment on this: The create step does have an assertion that's relevant to the test, so I don't really view it as pure setup. Keeping it as part of the test – the way it is right now – makes the most sense to me.

Do you think this part looks ready to merge now?

Reply:
Yep, LGTM!